src/lib/components/wizard.component.ts

```diff
@@ -216,7 +187,9 @@ export class WizardComponent implements AfterContentInit, OnChanges {
 
     // initialize the model
     this.updateWizardSteps(this.wizardStepsQueryList.toArray());
-    this.updateNavigationMode(this.navigationMode);
+    if (!this._navigation) {
+      this._navigation = new ConfigurableNavigationMode();
+    }
```

Comment:
These lines cover the case when `[awNavigationMode]` directive is absent.

Reply:
Why not use 
```typescript
public _navigation: NavigationMode = new ConfigurableNavigationMode();
```
instead?

Reply:
Right. Done in 65a64a7.